patches/minecraft/net/minecraft/world/entity/animal/Wolf.java.patch

```diff
@@ -9,3 +18,13 @@
                 this.m_21828_(p_30412_);
                 this.f_21344_.m_26573_();
                 this.m_6710_((LivingEntity)null);
+@@ -400,8 +_,7 @@
+    }
+ 
+    public boolean m_6898_(ItemStack p_30440_) {
+-      Item item = p_30440_.m_41720_();
+-      return item.m_41472_() && item.m_41473_().m_38746_();
```

Comment:
Don't change the first line or the `item#isEdible` call as that just makes it a larger patch, only change the call to `getFoodProperties`